### PR TITLE
Integration w frontend

### PR DESCRIPTION
**Pull request by benwubenwu**
i hope i did this rebase/pull request right

### CODE REVIEW

File: audiopedia/audios/schema.py [hunk 1]
```diff
@@ -24,6 +24,11 @@ class Meta:
         model = Topic
         fields=("title", "index", "audio_url", "active", "published", "playlists")
 
+class LanguageInput(graphene.InputObjectType):
```
[the remark this answers is not in the document]
Comment (benwubenwu) [reply audiopedia/audios/schema.py:27]: Yeah the only stuff that really needs to be merged is the CORS stuff in the settings file I believe. All the other mutations or models stuff from master is more up to date I believe.

[the remark this answers is not in the document]
Comment (BachToTheFuture) [reply audiopedia/audios/schema.py:27]: Yup. If you can somehow just have the CORs stuff on your PR it should be good!

[the remark this answers is not in the document]
Comment (BachToTheFuture) [reply audiopedia/audios/schema.py:27]: I also recently updated the master branch with some stuff for the docs so you might want to just make a new branch based off of the master branch and just copy over what you have for CORs.

**Comment by BachToTheFuture** [comment]
changes already merged with a different pull request!

**Pull request closed without merging**